Library/Tests/TestHelpers/DispatchTimeInterval-Extensions.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+extension DispatchTimeInterval {
```

Comment:
I've commented about `DispatchTimeInterval` a bit, but it's a cool new enum in swift 3 that unifies (milli/micro/nano)seconds. this file just contains a few lil extensions that i found useful for our tests.